src/components/pages/SponsorsPage.jsx

```diff
@@ -4,11 +4,12 @@ import Sponsors from 'components/elements/Sponsors'
 
 function SponsorsPage() {
   return (
-    <div className="container mx-auto p-10">
-      <h1 className="text-2xl font-bold">Thank you to our Sponsors!</h1>
-      <p className="text-lg pl-0.5 pt-2 pb-2">
+    <div className="container mx-auto pt-5 md:pt-14 pb-56 dark:text-[#F7FAFC]">
+      <h1 className="hidden md:block text-2xl font-bold pb-6">Thank you to our Sponsors!</h1>
+      <h1 className="visible md:hidden text-2xl pb-5">Sponsors</h1>
```

Comment:
only 1 h1 per page, this needs to be an h2

Reply:
Corrected!